### DISCUSSION TITLE
GPU utilization of WebRTC protocol

### DISCUSSION THREAD

**Discussion by GyudongPark** [bluenviron/mediamtx#3654] [Questions]
### Question

If some other process utilize GPU 100% (checking by nvidia-smi), then can it affect performance of WebRTC like latency?

**Comment by aler9** [comment]
Hello, MediaMTX is just in charge of routing streams from one end to another, and it doesn't make use of the GPU. The entity that might make use of the GPU is the browser, when the decoder associated with the codec of the stream is GPU-based (for instance, H265 and AV1 on Windows). In this case, video might be displayed late or corrupted.

I think you can disable interaction with the GPU by tuning settings:
https://support.udemy.com/hc/en-us/articles/22543012795799-Troubleshooting-How-to-Disable-Hardware-Acceleration-on-Your-Browser

Some browsers might be able to play streams without GPU, solving the issue.

[a comment by an automated account is not shown]
